packages/react-native-renderer/src/ReactFabricGlobalResponderHandler.js

```diff
@@ -13,7 +13,8 @@ import {UIManager} from 'react-native/Libraries/ReactPrivate/ReactNativePrivateI
 const ReactFabricGlobalResponderHandler = {
   onChange: function(from: any, to: any, blockNativeResponder: boolean) {
```

Comment:
Is there a plan to flow type this?

Reply:
Surprisingly "handles" like this seem to be consistently any-typed in the ReactNativeRenderer and in the Fabric renderer. So I guess I'm just going to leave as any for now? Seems odd